libkbfs/kbfs_ops_test.go

```diff
@@ -2952,35 +2952,30 @@ func TestKBFSOpsBackgroundFlush(t *testing.T) {
 		t.Fatalf("Couldn't create file: %+v", err)
 	}
 
-	data := []byte{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}
-	if err := kbfsOps.Write(ctx, nodeA, data, 0); err != nil {
-		t.Errorf("Got error on write: %+v", err)
-	}
+	ops := getOps(config, rootNode.GetFolderBranch().Tlf)
+	oldPtr := ops.nodeCache.PathFromNode(nodeA).tailPointer()
 
-	c := make(chan struct{})
-	observer := &testBGObserver{c}
-	config.Notifier().RegisterForChanges(
-		[]FolderBranch{rootNode.GetFolderBranch()}, observer)
+	staller := NewNaïveStaller(config)
```

Comment:
I added that to experiment and the plan was keep it until somebody complains. So I'm OK with changing it to `NaiveStaller`. :-) I'll make the PR later if @strib you don't want to deal with it in this PR